lib/vttcue.js

```diff
@@ -258,5 +304,5 @@
     return WebVTT.convertCueToDOMTree(window, this.text);
   };
 
-  root.VTTCue = root.VTTCue || VTTCue;
+  root.VTTCue = VTTCue || root.VTTCue;
```

Comment:
@RickEyre do you have any ideas how to feature detect whether the native VTTCue is recent enough? https://github.com/mozilla/vtt.js/issues/331#issuecomment-62245442
Going to work on making this configurable/optional in a separate PR.